### DISCUSSION TITLE
Uyarılan Bölgelerde Riskli Bina Tespit App'i

### DISCUSSION THREAD

**Discussion by bakicpp** [acikkaynak/deprem-yardim-projesi#34] [Ideas]
Uzmanlar tarafından uyarılan bölgelerde yer alan binaların kullanıcılar tarafından bildirilerek kamuoyu oluşturulması ve oluşturulan kamuoyu baskısı ile yetkili birimlerin riskli binaları yapılandırmasının hızlandırılmasını içerir.

**Comment by sercangoger** [comment]
app olmasına gerek yok. sayfa da hızlı bir şekilde sayfa oluşturulup deprem.io altında yapılabilir. daha hızlı olacaktır.

**Reply by bakicpp** [reply]
katılıyorum, daha hızlı olacaktır o şekilde

**Comment by ramazansancar** [comment]
Merhaba @bakicpp,
Şu an "App: Uydu" adında projemiz var. Ona dahil edilebilir.

**Reply by corupta** [reply]
uyduda goruntuler uzerinden calismaya kanalize hareket ediyoruz, insanlardan gelen yazili verilere iliskin bir projeyse ayri kanalda olmasi daha iyi olur. ek olarak uydu kanalinda hasarli bina tespiti verilerini olusturdukca paylasabiliriz o kanalla da

**Reply by corupta** [reply]
uydu goruntusunden binanin hasarli oldugu bile zor belli oluyor, riskli oldugunu anlayabilecegimizi sanmiyorum onun icin binalarin uzmanlarca incelenmesi lazim heralde, insaat muh arkadaslardan bilgi almak mantikli olabilir riski tanimlama konusunda

**Reply by bakicpp** [reply]
aslında bu projenin uyduyla pek bağlantısı yok. binanın riskli olup olmadığı bölgede yaşayan kullanıcılar tarafından sisteme bildirilecek. sistem de verilerini yetkili kurumlarla paylaşıp "burası fazla istek almış, buraları kontrol edin" anlamında kurumları uyararak yapılandırmaya teşvik edecek. aslında bu projenin en etkili yanı kamuoyu baskısı kullanmak olacak. @ramazansancar

**Comment by corupta** [comment]
Binanin riskli oldugunu neye gore tanimliyorsunuz?

**Reply by bakicpp** [reply]
binanın riskli olup olmadığı bölgede yaşayan kullanıcılar tarafından sisteme bildirilecek. sistem de verilerini yetkili kurumlarla paylaşıp "burası fazla istek almış, buraları kontrol edin" anlamında kurumları uyararak yapılandırmaya teşvik edecek. aslında bu projenin en etkili yanı kamuoyu baskısı kullanmak olacak.
